Xamarin.Forms.Core/IElementConfiguration.cs

```diff
@@ -1,7 +1,7 @@
 
 namespace Xamarin.Forms
 {
-	public interface IElementConfiguration<out TElement> where TElement : Element
+	public interface IElementConfiguration<TElement> where TElement : Element
```

Comment:
this is a potential breaking change, but I think it's only used by platforms

Reply:
Rolled this back.